Userland/Utilities/passwd.cpp

```diff
@@ -95,6 +95,13 @@ int main(int argc, char** argv)
 
         auto new_password = Core::get_password("New password: ");
         auto new_password_retype = Core::get_password("Retype new password: ");
+
+        if (new_password.value().is_empty() && new_password_retype.value().is_empty()) {
```

Comment:
Neither of the above values have been error checked.

Reply:
Changes made.